language/move-stdlib/src/lib.rs

```diff
@@ -146,3 +151,25 @@ pub fn build_nursery_doc(output_path: &str) {
         false,
     )
 }
+
+pub fn build_error_code_map(output_path: &str) {
```

Comment:
@tzakian can you double check this

Reply:
LGTM!

Reply:
I wonder if we should also include the nursery modules